hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/ChildFeeContextImpl.java

```diff
@@ -43,16 +43,19 @@ public class ChildFeeContextImpl implements FeeContext {
     private final HandleContextImpl context;
     private final TransactionBody body;
     private final AccountID payerId;
+    private final boolean computeFeesAsInternalDispatch;
 
     public ChildFeeContextImpl(
             @NonNull final FeeManager feeManager,
             @NonNull final HandleContextImpl context,
             @NonNull final TransactionBody body,
-            @NonNull final AccountID payerId) {
+            @NonNull final AccountID payerId,
+            final boolean computeFeesAsInternalDispatch) {
```

Comment:
I know this is just temporary (for diff testing) but it still would have been nice if this variable was the same "direction" as `ComputeDispatchFeesAsTopLevel`, i.e., `ComputeDispatchFeesAsTopLevel.YES` == `true` here.  Instead, it's the reverse.  Unfortunate mental gymnastics.

(You see the direction change over at `HandleContextInfo` L529.)

Reply:
Agreeed.